### ISSUE TITLE
TTL Queue Log

### ISSUE THREAD

**Issue by erasaur** [Meteor-Reaction/Herald#54]
The queue log can rack up quite a few documents over time. I don't recall them being particularly important, so maybe we can just expire them after some time?

**Comment by Kestanous** [comment]
Agreed. Do you think 24 hours is reasonable? We should also look at #21. Heavy Herald usage currently causes far to much pollution. I wonder if we should delete by default and let users add more time/disable on requests.

**Comment by erasaur** [comment]
24 hours seems good. And I agree, we should let the user configure this number for their app. For me personally, I don't think I even need as much as 24 hours.

**Comment by Kestanous** [comment]
Well shoot lets set it to an hour then. That will work for dev usage and users will want to check their configs before they go production anyway. Will will just need to let the current users know of the change.

**Comment by erasaur** [comment]
Sure. I think the README is due for some updating anyway.

**Comment by Kestanous** [comment]
Definitely. Your magic nesting curriers need some spot light as well ^_^

**Comment by erasaur** [comment]
Haha of course :) I think we ought to also mention some things about the TTL on the notifications themselves. Because we're using an index, we should probably warn users to take care in changing the options too frequently, since there could be some negative side effects (e.g slowness in building the index, leftover indexes). I can elaborate a little more on this in the README.

**Comment by Kestanous** [comment]
Hmm yes. Please do update as you see fit, ping me if you need anything. I am working hard on another package right now (https://github.com/Meteor-Reaction/Candid) but I don't want to neglect Herald.

**Comment by erasaur** [comment]
I noticed that you were working on that -- very useful stuff in the making. Maybe I'll see if I can chip in to Candid as well :) I'll go ahead and update the README with a quick notice right now. I'll add more later when I have some time.

**Comment by Kestanous** [comment]
@erasaur another point I was thinking about but the introduction paragraphs read like a legal doc, which I don't feel is very attractive. I have no attachment to that or any other wording/formatting, so if you want change a will. If you don't I likely will soon ^_^
